skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/model/Dto.kt

```diff
@@ -17,7 +17,9 @@ object Dto {
         val redigertBrev: Edit.Letter,
         val redigertBrevHash: EditLetterHash,
         val saksbehandlerValg: SaksbehandlerValg,
-    )
+    ) {
+        fun erVedtaksbrev() = info.vedtaksId != null
```

Comment:
Det er en implisitt konsekvens at brev har vedtaksId for vedtaksbrev. Jeg tror at det teknisk er mulig å ende opp med brev uten vedtaksId for vedtaksbrev, siden jeg tror ikke vi validerer det noe sted. Brev-uten-data er et eksempel, siden man ikke trenger vedtaksId for å bygge opp EmptyBrevbakerdata.

Reply:
Var det eg var stygt redd for, ja. Har vi ein tryggare mekanisme? Viss ikkje er det vel på tide å innføre.

Reply:
`templateDescription.metadata.brevtype` skal være `VEDTAKSBREV` for alle brev som skal attesteres.